src/agent/centralized_configuration/include/centralized_configuration.hpp

```diff
@@ -18,8 +18,11 @@ namespace centralized_configuration
     public:
         using SetGroupIdFunctionType = std::function<void(const std::vector<std::string>& groupList)>;
         using GetGroupIdFunctionType = std::function<std::vector<std::string>()>;
+        using SaveGroupIdFunctionType = std::function<void()>;
         using DownloadGroupFilesFunctionType =
             std::function<bool(const std::string& group, const std::string& dstFilePath)>;
+        using ValidateFileFunctionType = std::function<bool(const std::filesystem::path& configFile)>;
```

Comment:
For this we need to include `filesystem` header.

Reply:
Done!